### DISCUSSION TITLE
Localsend is blank when opened

### DISCUSSION THREAD

**Discussion by Awesome-GJK** [localsend/localsend#1186] [Discussion (English, International)]
I downloaded the latest compressed file and it works fine on my other two computers, but one of them is blank when opened. Does anyone know why?
![image](https://github.com/localsend/localsend/assets/127288468/4f2f37b2-e66f-48ce-9be3-64900491c3f3)

**Comment by Tienisto** [comment]
Is it Windows 8?

**Comment by Awesome-GJK** [comment]
windows 10

Tien Do Nam ***@***.***> 于2024年3月6日周三 18:43写道：
